source/common/http/utility.cc

```diff
@@ -121,6 +121,47 @@ std::string Utility::parseCookieValue(const HeaderMap& headers, const std::strin
   return state.ret_;
 }
 
+std::string Utility::makeSetCookieValue(const std::string& key, const std::string& value,
+                                        const std::chrono::seconds max_age) {
+  return fmt::format("{}=\"{}\"; Max-Age={}", key, value, max_age.count());
```

Comment:
Is there any situation in which we might want to ensure that `value` is correctly escaped? Probably not today, but seems like something dangerous if `value` could come from the user.

Reply:
Agreed; I've added a comment to the function declaration comment.